protocol/simplex-messaging.md

```diff
@@ -360,6 +360,8 @@ The clients can optionally instruct a dedicated push notification server to subs
 - `subscribeNotifications` (`"NSUB"`) - see [Subscribe to queue notifications](#subscribe-to-queue-notifications).
 - `messageNotification` (`"NMSG"`) - see [Deliver message notification](#deliver-message-notification).
 
+[`SEND` command](#send-message) includes the notification flag to instruct SMP server whether to send the notification - this flag is forwarded to the recepient inside encrypted envelope, together with the timestamp and the message body, so even if TLS is compromised this flag cannot be used for traffic correlation.
```

Comment:
meta is outside encrypted envelope in `messageWithMeta` variant of `message` below - maybe it's better to mention here it's old protocol or in`MSG` abnf it's for backwards compatibility

Reply:
updated